mu-trees/addon/resolvers/glimmer-wrapper/index.js

```diff
@@ -81,7 +81,15 @@ const Resolver = GlobalsResolver.extend({
     this._glimmerResolver = new GlimmerResolver(this.config, this.glimmerModuleRegistry);
   },
 
-  normalize: null,
+  normalize(specifier) {
```

Comment:
Since this is only needed for debug builds, can you move the functionality to be added in debug builds only?

```js
import { DEBUG } from '@glimmer/env';

class Resolver {
  // existing stuff
}

if (DEBUG) {
  Resolver.prototype.normalize = function(specifier) {
    // implementation from this PR
  }
}
```